src/sdk-reference/js/6/extend-sdk/index.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: sdk.html.hbs
+title: Extends the SDK
+description: Extends the SDK
+order: 410
+---
+
+# Extends the SDK with a custom SDK controller
+
+It is possible to extend the SDK's functionality by adding new controllers.  
+
+These controllers generally correspond to [custom controllers created in a plugin]({{ site_base_path }}plugins/1/essentials/controllers) but not necessarily. Thus, it is possible to use the actions of its plugin in the SDK in the same way as the other actions of the Kuzzle API.  
```

Comment:
I don't understand the sentence starting with "Thus..."
What do you mean exactly?

As a side note, I think that matching an API extension made by a plugin is the main usage for that class, and I think that this is how this should be documented.

Reply:
I mean that you can use your plugin custom action in the same way as core API actions. (eg: `kuzzle.myController.myAction()`)

Reply:
I find your new version clearer :+1: